Sources/Application/Routes/MarkdownRoutes.swift

```diff
@@ -22,19 +22,14 @@ func initializeMarkdownRoutes(app: App) {
 
     app.router.setDefault(templateEngine: KituraMarkdown())
 
-    app.router.get("/docs") { _, response, next in
-        response.headers["Content-Type"] = "text/html"
-        try response.render("/docs/index.md", context: [String:Any]())
-        response.status(.OK)
-        next()
-    }
-
     app.router.get("/docs/*") { request, response, next in
         response.headers["Content-Type"] = "text/html"
-        if request.urlURL.path != "/docs/" {
+        if request.urlURL.path == "/docs/" || request.urlURL.path == "/docs" {
```

Comment:
This route will not match "/docs" and so the second check will never fail. You can either just remove it or remove it and re add the route above.